src/PKSim.UI/Views/PopulationAnalyses/BaseChartView.cs

```diff
@@ -92,32 +105,34 @@ protected override void OnDragDrop(DragEventArgs e)
       private void onObjectHotTracked(HotTrackEventArgs e)
       {
          var series = e.Series();
+         _latestTrackedCurvedData = null;
+         _latestSeriesPoint = null;
          if (series == null)
          {
             hideToolTip(e);
             return;
          }
 
          var diagramCoordinates = _chartControl.DiagramCoordinatesAt(e);
-         var point = e.HitInfo.SeriesPoint;
+         _latestSeriesPoint  = e.HitInfo.SeriesPoint;
          SuperToolTip superToolTip = null;
 
          var observedCurveData = _presenter.ObservedCurveDataFor(diagramCoordinates.Pane.Name, series.Name);
          if (observedCurveData != null)
          {
-            superToolTip = getSuperToolTipFor(observedCurveData, point, diagramCoordinates);
+            superToolTip = getSuperToolTipFor(observedCurveData, _latestSeriesPoint, diagramCoordinates);
          }
 
          if (superToolTip == null)
          {
-            var curveData = _presenter.CurveDataFor(diagramCoordinates.Pane.Name, series.Name);
-            if (curveData == null)
+            _latestTrackedCurvedData  = _presenter.CurveDataFor(diagramCoordinates.Pane.Name, series.Name);
+            if (_latestTrackedCurvedData == null)
```

Comment:
In the view, I am keeping track of the last serie and point that was tracked so that I can retrieve the Box for the extraction